content/en/docs/concepts/services-networking/service.md

```diff
@@ -666,6 +666,22 @@ The value of `spec.loadBalancerClass` must be a label-style identifier,
 with an optional prefix such as "`internal-vip`" or "`example.com/internal-vip`".
 Unprefixed names are reserved for end-users.
 
+#### Specifying IPMode of load balancer status {#load-balancer-ip-mode}
+
+{{< feature-state for_k8s_version="v1.29" state="alpha" >}}
+
+Starting with Kubernetes 1.29 in alpha, a feature gate named `LoadBalancerIPMode` allows you to set the `.status.loadBalancer.ingress.ipMode` 
```

Comment:
```suggestion
Starting as Alpha in Kubernetes 1.29, a feature gate named `LoadBalancerIPMode`
allows you to set the `.status.loadBalancer.ingress.ipMode` 
```

Reply:
Even better: hyperlink “feature gate” to https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/